src/components/Constants.js

```diff
@@ -138,7 +138,8 @@ export const PAGE_ESSENTIALS = {
     key: "profile",
     routes: [
       "teams.stats",
-      "communities.list"
+      "communities.list", 
```

Comment:
@frimpongopoku can you resolve this comment so we move the changes forward?

Reply:
@abdullai-t because we need the list of communities for that page...? Also, the only change there is a comma, to make way for the real change "data.carbonEquivalency.get" below.